08-transport.Rmd

```diff
@@ -269,16 +289,18 @@ desire_lines = od2line(od_inter, zones)
 ```
 
 The first two lines of the preceding code chunk split the `od` dataset into two mutually exclusive objects, `od_intra` (which only contains OD pairs representing intrazone trips) and `od_inter` (which represents interzonal travel).
-The third line generates a geographic object `desire_lines` (of class `sf`) that allows the interzone trips to be visualized geographically, using plot commands such as that provided in the command below.
-An illustration of the results are presented in Figure \@ref(fig:desire) (we will cover the visualization methods that produced this plot in a subsequent chapter).
-The map shows the city center dominate transport patterns in the region, suggesting policies should be prioritized there, although a number of peripheral sub-centers can also be seen.
+The third line generates a geographic object `desire_lines` (of class `sf`) that allows a subsequent geographic visualization of interzone trips.
+An illustration of the results are presented in Figure \@ref(fig:desire) (we will cover the visualization methods that produced this plot in a subsequent chapter) <!-- to do: add reference to chapter -->.
+The map shows the city center dominates transport patterns in the region, suggesting policies should be prioritized there, although a number of peripheral sub-centers can also be seen.
+Next it would be interesting to have a look at the distribution of interzonal modes, e.g., between which zones is cycling the least, the most common means of transport. <!-- maybe an idea for an exercise? -->
 <!-- These include Bradford Stoke to the North and Portishead to the West. -->
 
 ```{r, eval=FALSE}
 plot(desire_lines$geometry, lwd = desire_lines$all / 500)
 ```
 
-```{r desire, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Desire lines representing trip patterns in the Bristol Travel to Work Area. The 4 black lines represent interzonal OD pairs in Table 8.1."}
+<!-- might be a bit confusing that the four black lines are shown but not created in the code chunk, maybe add clarification -->
+```{r desire, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Desire lines representing trip patterns in the Bristol Travel to Work Area. The four black lines represent interzonal OD pairs in Table 8.1."}
```

Comment:
might be a bit confusing that the four black lines are shown but not created in the code chunk, maybe add clarification